wini/scripts/sync-template.sh

```
@@ -31,5 +33,19 @@
-git cherry-pick "$(yq ".last_commit_hash" <<< "$origin")"..wini-template/"$(yq ".branch" <<< "$origin")"
-git remote remove wini-template
+
+# Check for coherent last_commit_hash
+if ! git merge-base --is-ancestor "$last_commit_hash" "wini-template/$branch"; then
+    error 'Invalid `last_commit_hash`: doesn'"'t exists in remote wini-template/$branch"
+    git remote remove wini-template
+fi
+
+if ! git cherry-pick "$last_commit_hash"..wini-template/"$branch"; then
+    git remote remove wini-template || true
+    error 'Cherry-pick failed; please resolve conflicts and after that, do `wini sync-commit-hash`'
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)